compiler/frontend/src/org/jetbrains/kotlin/diagnostics/PositioningStrategies.kt

```diff
@@ -756,6 +756,7 @@ object PositioningStrategies {
                 }
                 is KtCallExpression -> return mark(element.calleeExpression ?: element)
                 is KtConstructorDelegationCall -> return mark(element.calleeExpression ?: element)
+                is KtSuperTypeCallEntry -> return mark(element.calleeExpression)
```

Comment:
Just to make sure, don't we need safe guard `... ?: element` like others above?

Reply:
Yep, `calleeExpression` is marked @NotNull in KtSuperTypeCallEntry.